src/main/host/shd-network-interface.h

```diff
@@ -7,10 +7,10 @@
 #ifndef SHD_NETWORK_INTERFACE_H_
 #define SHD_NETWORK_INTERFACE_H_
 
-#include "shadow.h"
-
 typedef struct _NetworkInterface NetworkInterface;
 
+#include "shadow.h"
```

Comment:
I think the usual convention is for #includes to be the first thing after the include-guard? If there's a specific reason why NetworkInterface needs to be defined first, probably worth a comment?